.coveragerc

```diff
@@ -3,4 +3,4 @@ branch = True
 omit = */tests/*,**/contract_tests.py
 
 [report]
-fail_under = 83
+fail_under = 86.5
```

Comment:
Not a maximum.

![image](https://user-images.githubusercontent.com/3982807/70957616-88430e80-202b-11ea-89cc-611dadaa59c0.png)

Reply:
local maximum, brah